test/fixtures/combined_vancouver_queens.json

```diff
@@ -892395,8 +892389,7 @@
     "_type": "_doc",
     "data": {
       "name": {
-        "default": "IPOH Asian House"
-      },
+        "default": "IPOH Asian House" },
```

Comment:
non-issue: this is weirdly indented

Reply:
oh, oops! My fault! I had my hand at editing the fixture manually since there were only a few changes.

Should have left it to the machines.

Reply:
haha, it's really not an issue, I just noticed it a few times in the PR, like the `trim( tags[key]))` catches my eye.

the styling used to be more hit-and-miss, I've been using autoformatting in my editor for a while now which hopefully fixes a bunch of the little ones.

at some point I'd still love to fully adopt [standardJS](https://standardjs.com/).